app/src/layouts/cards/components/card.vue

```diff
@@ -132,7 +153,26 @@ export default defineComponent({
 			return props.modelValue.includes(props.item[props.itemKey]) ? 'check_circle' : 'radio_button_unchecked';
 		});
 
-		return { imageSource, svgSource, type, selectionIcon, toggleSelection, handleClick, imgError };
+		onMounted(() => {
+			resizeObserver.value = new ResizeObserver(() => {
```

Comment:
I'm a little worried about using a ResizeObserver instance per card on the layout, it can render up to a 1000 at a time, which might severely hinder page performance 🤔

Reply:
When and why do these get resized? When using the size-selector and widow onresize?

Isn't possible to only observe the ones that are in view? Maybe in combination with some lazyload?

Reply:
Not sure what it's currently used for! Just seeing heavier logic in a component that is rendered 1k times gave me a bit of a 🚨🚩 moment 😁

Reply:
For sure! Luckily Vue 3 handles that fairly gracefully, so that's a bit of a tradeoff between performance and ease of maintenance/customization. That being said, we still gotta be careful with bigger known potential perf issues 🙂 

> edit: @rijkvanzanten was trying to send a link via PM =p no success, and understandable

?